### ISSUE TITLE
anndata "obs" are cells, not genes

### ISSUE THREAD

**Issue by jamestwebber** [YosefLab/scVI#84]
Had some issues loading our data and realized that there was a transpose issue. Looking at [the code](https://github.com/YosefLab/scVI/blob/master/scvi/dataset/anndata.py#L24), I believe there's a bug. The `obs` of an anndata object should be the cells, not the genes. The genes are `var`. So on the plus side for you, no transpose is necessary for loading.

Correct me if I'm wrong, as I may have just confused myself...

**Comment by imyiningliu** [comment]
Hi, we tested the code using [TabulaMuris data](https://s3.amazonaws.com/czbiohub-tabula-muris/TabulaMuris.h5ad.zip) and I think they used cells as `obs` and genes as `var`. Is it more common to use `obs` as cells? I can add an extra parameter and set the default to be `obs=cell` to the class constructor. Thanks for catching that! 😊

**Comment by jamestwebber** [comment]
That explains it! I made that file and I did it wrong–which led to it working fine with scVI but then our newer files were incompatible. The problem was I didn't search for any examples when I used `anndata.read_mtx` and assumed it was in the right orientation. [This notebook](https://github.com/dpcook/fun_analysis/blob/master/tabula_muris/mouse_atlas_scanpy.ipynb) shows them transposing the result.

I'll update the h5ad file on S3 (you might be the only group to use it so far, since no one else has said anything).
